src/components/WebexLocalMedia/WebexLocalMedia.jsx

```diff
@@ -11,6 +11,7 @@ import {
   useMeeting,
   useStream,
 } from '../hooks';
+import Banner from '../generic/Banner/Banner';
```

Comment:
the generic components should be above the more specific components (eg, above WebexAvatar)